docs/cli/testkube_abort.md

```diff
@@ -12,18 +12,6 @@ testkube abort <resourceName> [flags]
   -h, --help   help for abort
 ```
 
-### Options inherited from parent commands
```

Comment:
I'm not sure why it suddenly hides these but I don't want to lose a lot of time to investigate. I guess this is alright for now?

Reply:
let's create a follow-up ticket for this though - doesn't seem right that all inherited params are suddenly gone..